site/content/docs/main/api-types/schedule.md

```diff
@@ -143,7 +143,9 @@ spec:
           post:
             # Same content as pre above.
 status:
-  # The current phase of the latest scheduled backup. Valid values are New, FailedValidation, InProgress, Completed, PartiallyFailed, Failed.
+  # The current phase of the latest scheduled backup.
+  # Valid values are New, FailedValidation, InProgress, WaitingForPluginOperations,
+  # WaitingForPluginOperationsPartiallyFailed, Completed, PartiallyFailed, Failed.
   phase: ""
```

Comment:
@sseago I believe this phase is the current phase of the schedule and not backup. Schedule phase is different from backup phase https://github.com/vmware-tanzu/velero/blob/main/pkg/apis/velero/v1/schedule_types.go#L67

Reply:
@shubham-pampattiwar Oh, you're right The doc is wrong here -- it says that phase is the phase of the backup, but it's not, it's hte schedule phase. I'll update the comments to reflect this.